Add snap connection commands for Ubuntu Core (#15)
canonical/chip-tool-snap: README.md

sudo snap connect chip-tool:bluez
```

> **Note**  
> On **Ubuntu Core**, the `avahi-control` and `bluez` interfaces are not provided by the system.
> These interfaces should be consumed from other snaps, such as the [Avahi](https://snapcraft.io/avahi) and [BlueZ](https://snapcraft.io/bluez) snaps.
> 

> > To install the snaps, and establish connections for the `avahi-control` interface from the `avahi` snap, and the `service` interface from the `bluez` snap, run:

Reviewer: avahi-control or avahi-observe?

> > On **Ubuntu Core**, the `avahi-control` and `bluez` interfaces are not provided by the system.
> > These interfaces should be consumed from other snaps, such as the [Avahi](https://snapcraft.io/avahi) and [BlueZ](https://snapcraft.io/bluez) snaps.
> > 
> > To install the snaps, and establish connections for the `avahi-control` interface from the `avahi` snap, and the `service` interface from the `bluez` snap, run:

Reviewer: Too wordy, consider:
```suggestion
> On **Ubuntu Core**, the `avahi-control` and `bluez` interfaces are not provided by the system.
> These interfaces are provided by other snaps, such as the [Avahi](https://snapcraft.io/avahi) and [BlueZ](https://snapcraft.io/bluez) snaps. To install the snaps and connect to the interfaces, run:
```
